src/printers/logs.ts

```diff
@@ -0,0 +1,11 @@
+import { LogList, LogApiResource } from '@twilio-labs/serverless-api';
+import { LogsConfig } from '../config/logs';
+import { logger } from '../utils/logger';
+
+export function printLogs(result: LogApiResource[], config: LogsConfig) {
+  result.forEach(printLog);
+}
+
+export function printLog(log: LogApiResource) {
+  logger.info(`[${log.level}][${log.date_created}]: ${log.message}`);
```

Comment:
Let's future proof this. #62 is calling for the same support as the Twilio CLI in terms of output. Let's add support for an `--output` or `-o` flag that supports at least `json` as another option where each line will be it's own line of JSON. That way you could pipe it into a command to split by line and then run `jq` on it to filter for certain things. 
If you want you could add `tsv` support as well but I think `json` is for now the only thing.

Reply:
Ah yeah, I actually meant to do this. Being able to output JSON particularly so that the whole object can be piped to something else to render differently.

Reply:
I've added this now.